src/couch/src/couch_util.erl

```diff
@@ -795,3 +808,20 @@ hmac(Alg, Key, Message) ->
     crypto:hmac(Alg, Key, Message).
 
 -endif. % -ifdef(OTP_RELEASE)
+
+unquote(Binary) when is_binary(Binary) ->
+    unquote(binary_to_list(Binary));
+unquote(String) ->
+    qs_revdecode(lists:reverse(String)).
+
+qs_revdecode(S) ->
+    qs_revdecode(S, []).
+
+qs_revdecode([], Acc) ->
+    Acc;
+qs_revdecode([$+ | Rest], Acc) ->
```

Comment:
We don't need this clause since this case would be handled on line 826.

Reply:
Fixed in https://github.com/apache/couchdb/pull/3638/commits/10061e63a196c0851cd0894849034ee501858111.